app/src/main/java/org/schabi/newpipe/fragments/EmptyFragment.java

```diff
@@ -6,9 +6,11 @@
 import android.view.ViewGroup;
 
 import androidx.annotation.Nullable;
+import androidx.compose.ui.platform.ComposeView;
 
 import org.schabi.newpipe.BaseFragment;
 import org.schabi.newpipe.R;
+import org.schabi.newpipe.ui.emptystate.EmptyStateUtil;
 
 public class EmptyFragment extends BaseFragment {
```

Comment:
The entire fragment could be implemented using Compose directly, see https://developer.android.com/reference/kotlin/androidx/fragment/compose/package-summary#(androidx.fragment.app.Fragment).content(kotlin.Function0)

Reply:
OK. This PR is to have a single purpose. Since applying this API requires firstly migrating EmptyFragment to kotlin, I can start another PR purely for the migration without mixing it into one.